pysmt/smtlib/solver.py

```diff
@@ -96,6 +96,9 @@ def __init__(self, args, environment, logic, LOGICS=None, **options):
             self.solver_stdout = TextIOWrapper(self.solver.stdout)
 
         # Initialize solver
+        if "produce_interpolants" in options and options["produce_interpolants"]:
+            self.set_option(":print-success", "true")
```

Comment:
is the `:print-success` needed for the interpolation work or was this for debugging?